enhancements/network/allow-mtu-changes.md

```diff
@@ -0,0 +1,207 @@
+---
+title: allow-mtu-changes
+authors:
+  - "@juanluisvaladas"
+  - "@jcaamano"
+reviewers:
+  - "@danwinship"
+  - "@dcbw"
+  - "@knobunc"
+  - "@msherif1234"
+approvers:
+  - TBD
+creation-date: 2021-10-07
+last-updated: 2021-10-07
+status: provisional
+---
+
+# Allow MTU changes
+
+This covers adding the capability to the cluster network operator of changing
+the MTU post installation.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Customers may need to change the MTU post-installation. However these changes
+aren't trivial and may cause downtime, hence the CNO currently forbids them.
+
+We propose a procedure that will be launched on demand. These procedure will
+run pods on every node of the cluster and make the necessary changes in an
+ordered and coordinated manner with a service disruption within the least
+possible time and hopefully within TCP timeout (best effort).
+
+## Motivation
+
+While cluster administrators usually set the MTU correctly during the
+installation sometimes they need to change them afterwards for reasons such as
+changes in the underlay or because they were set incorrectly at install time.
+
+### Goals
+
+* Allow to change MTU post install on OVN Kubernetes.
```

Comment:
I think we had wanted to do this for openshift-sdn too?

Reply:
I think we do but I understood that not at the same time and not for 4.10 anyway so I did not cover it in this enhancement just because of time constraints and the fact that I don't know anything about it.